test/mocha/helpers.js

```diff
@@ -6,6 +6,7 @@
 const database = require('bedrock-mongodb');
 
 exports.isRegistration = result => {
+  // FIXME: add proper result checking instead of a console log.
```

Comment:
Will this be addressed in this PR? If not, it should have a link to an issue.

Reply:
I did create an issue for this since it's unrelated to this PR: https://github.com/digitalbazaar/bedrock-tokenization/issues/69